docs_src/security/Ch-DelayedStartServices.md

```diff
@@ -20,6 +20,32 @@ in order to use delayed start services.
 
 ## How to Enable (Docker)
 
+### Enable Custom Application or Device Services (Optional)
+If using EdgeX with custom Application or Device services in Secure mode, first generate a docker-compose.yml file by running the following command from edgex-compose/compose-builder
+
+```shell
+$ make gen delayed-start
+```
+
+Open the generated docker-compose.yml file and set the SPIFFE_CUSTOM_SERVICES Environment variable. 
+
+```yaml
+  security-spire-config:
+    ...
+    environment:
+      ...
+      SPIFFE_CUSTOM_SERVICES: <custom-service> <custom-service-2>
+```
```

Comment:
could you mention something about delimiter for custom-services (i.e. the white space as the delimiter for more than one custom services)? it's no biggie but that way is clear